textBasedAdventure.js

```diff
@@ -1,5 +1,321 @@
-const readline1 = require('readline-sync')
+const readline = require("readline-sync");
 
-let nameInput = readline.question("Enter your name: ")
+let nameInput = readline.question("Enter your name: ");
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+console.log(`Hello ${nameInput}!  Welcome to my game.`);
+
+const startGame = () => {
+  if (readline.keyInYNStrict("Do you want to play?")) {
+    console.log("Thank you for playing 'Choose your adventure' game.");
+    console.log(
+      "Throughout this game, you will get to make 10 choices to get to the ultimate adventure in the end."
+    );
+    console.log("Good luck!");
+    walkOrBike();
+  } else {
+    quitGame();
+  }
+};
+
+function walkOrBike() {
+    console.log("\n");
+    console.log("There is a forest path ahead of you. Would you like to walk or ride a bike?")
+  let res 
+  while(res !== "walk" && res !== "bike") {
+ res = readline
+    .question(
+       "Please enter walk or bike \n"
+    )
+    .trim()
+    .toLowerCase();
+
+  if (res === "walk") {
+    chooseAnimalWhileWalking();
+  } else if (res === "bike") {
+    chooseAnimalWhileBiking();
+  }
+}
+}
+function chooseAnimalWhileWalking() {
+  console.log("\n");
+    console.log(
+    "While walking you see several animals in the forest: rabbit, deer, tiger, lion, bear. "
+  );
+  let ind = readline.keyIn(
+    "Which one would you follow? (only the first letter) ",
+    { limit: 'rdtlb'}
+  ).toLowerCase();
+  if (ind === "r") {
```

Comment:
After we learn objects, you'll be able to map the letter to the word. Then you can just say something like
console.log(`Ok, you are following an ${animals[res]}`)) instead of doing all the conditions.